ports/python3/portfile.cmake

```
@@ -16,7 +16,7 @@ set(PATCHES
     0008-python.pc.patch
     0009-bz2d.patch
     0010-dont-skip-rpath.patch
-    0012-force-disable-curses.patch
+    0012-configure-no-libcrypt.patch
```

```suggestion
    0012-configure-no-libcrypt.patch #https://github.com/python/cpython/pull/28881
```
We can simplify the patch.

---

I'd missed 0012 somehow, I've updated this one to 0013